src/test/java/org/fundacionjala/pivotal/cucumber/stepdefinitions/api/RequestSteps.java

```diff
@@ -0,0 +1,85 @@
+package org.fundacionjala.pivotal.cucumber.stepdefinitions.api;
+
+import cucumber.api.java.en.And;
+import cucumber.api.java.en.Then;
+import cucumber.api.java.en.When;
+import io.restassured.response.Response;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+import org.fundacionjala.pivotal.cucumber.api.RequestManager;
+import org.fundacionjala.pivotal.cucumber.utils.StoreVariables;
+import org.fundacionjala.pivotal.cucumber.utils.Utils;
+
+/**
+ * Created by Administrator on 6/1/2017.
+ */
+public class RequestSteps {
+
+    private static final List<StoreVariables> STORE_VARIABLES = new ArrayList<>();
+    private Response response;
+
+    /**
+     * To get variables.
+     *
+     * @return List
+     */
+    public static List<StoreVariables> getVariables() {
+        return STORE_VARIABLES;
+    }
+
+    /**
+     * To send GET request.
+     *
+     * @param endpoint The endpoint to send
+     */
+    @When("^I request GET \"([^\"]*)\"$")
+    public void iRequestGETWith(final String endpoint) {
+        response = RequestManager.get(Utils.buildEndpoint(endpoint, STORE_VARIABLES));
+    }
+
+    /**
+     * The method allows to do a POST or PUT request based on the value of
+     * the parameter request String.
+     *
+     * @param request  of String type.
+     * @param endpoint of String type.
+     * @param body     is a Map.
+     */
+    @When("^I request \"(POST|PUT)\" \"([^\"]*)\" with:$")
+    public void iRequestPOSTWith(final String request, final String endpoint, final Map<String, String> body) {
+        String endpoint1 = Utils.buildEndpoint(endpoint, STORE_VARIABLES);
+        response = request.equals("POST") ? RequestManager.post(endpoint1, body)
+                : RequestManager.put(endpoint1, body);
+    }
+
+    /**
+     * To send DELETE request.
+     *
+     * @param endpoint The endpoint to send.
+     */
+    @Then("^I request DELETE \"([^\"]*)\"$")
+    public void requestDelete(final String endpoint) {
+        response = RequestManager.delete(Utils.buildEndpoint(endpoint, STORE_VARIABLES));
+    }
+
+    /**
+     * To return response.
+     *
+     * @return Response
+     */
+    public Response getResponse() {
+        return response;
+    }
+
+    /**
+     * To store value.
+     *
+     * @param variableName The variable's name to store
+     */
+    @And("^I stored as \\[([^\"]*)\\]$")
+    public void iStoredAsProject(final String variableName) {
```

Comment:
The method could be made more general for the other components of the project and does not remove